llvm/lib/Transforms/Instrumentation/DataFlowSanitizer.cpp

```diff
@@ -1546,7 +1546,8 @@ bool DataFlowSanitizer::runImpl(
   SmallPtrSet<Constant *, 1> PersonalityFns;
   for (Function &F : M)
     if (!F.isIntrinsic() && !DFSanRuntimeFunctions.contains(&F) &&
-        !LibAtomicFunction(F)) {
+        !LibAtomicFunction(F) &&
```

Comment:
what is about TSAN, ASAN, HWASAN, MSAN?

Reply:
Those are the ones already respecting the attribute. 

- https://clang.llvm.org/docs/MemorySanitizer.html#attribute-disable-sanitizer-instrumentation
- https://clang.llvm.org/docs/ThreadSanitizer.html#attribute-disable-sanitizer-instrumentation
- https://clang.llvm.org/docs/AddressSanitizer.html#disabling-instrumentation-with-attribute-no-sanitize-address